lib/galaxy/files/templates/examples/production_onedata.yml

```diff
@@ -0,0 +1,30 @@
+- id: onedata
+  version: 0
+  name: Onedata Storage
```

Comment:
potencjlanie do dogrania, chciałbym to zobaczyć w szerszym kontekście

Reply:
nie mam preferencji - storage może być bardziej explicite dla użytkowników sobie to potem wyklikujących, ale w sumie to chyba ogarną co znaczy `virtual filesystem`

Reply:
zostawmy jak jest